src/core/wee-command.c

```diff
@@ -5344,7 +5344,14 @@ COMMAND_CALLBACK(secure)
         }
         else
         {
+            if (temp_secure_passphrase == NULL || strcmp (temp_secure_passphrase, argv_eol[2]))
+            {
+                temp_secure_passphrase = strdup (argv_eol[2]);
+                gui_chat_printf (NULL, "Run the passphrase command again to confirm!");
```

Comment:
This string should probably be translatable, i.e. `_("Run the passphrase command again to confirm!")`.